src/languageSupport/languages/flux/monaco.flux.syntax.ts

```diff
@@ -12,7 +13,10 @@ async function addSyntax() {
     }),
   }))
 
-  window.monaco.languages.setLanguageConfiguration(LANGID, {
+  if (!self.monaco) {
+    self.monaco = monaco
+  }
+  self.monaco.languages.setLanguageConfiguration(LANGID, {
```

Comment:
window.monaco global object is no longer present.
Now use the self.monaco, for the web workers.